src/components/common/modal/Modal.module.css

```diff
@@ -0,0 +1,52 @@
+.overlay {
+  background-color: var(--overlay);
+  width: 100%;
+  height: 100%;
+  cursor: pointer;
+  transition: all 0.6s ease;
+}
+
+.modalContainer {
+  background-color: var(--white);
+  height: 72%;
+  width: 60%;
+  max-height: 55.2rem;
+  max-width: 81rem;
```

Comment:
WDYT about setting also `min-width` and `min-height`, to avoid it getting too small for smaller screens?

Reply:
Like it!